azure-pipelines.yml

```diff
@@ -98,6 +98,11 @@ strategy:
     #   skipCache: true
     #   needXmlstarlet: true
 
+    # 'git diff' validation
+    'git-diff':
+      image: 'ubuntu-20.04'
+      cmd: "./.ci/validation.sh git-diff"
```

Comment:
Execution independently is not going to find a diff.
You need to run it after "cmd: "mvn -e --no-transfer-progress verify""

Reply:
Changes Done